docs/use_cases/recSys-basic.ipynb

```diff
@@ -0,0 +1,3606 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "cb288679",
+   "metadata": {},
+   "source": [
+    "## Why do we build Recommender Systems?\n",
+    "\n",
+    "Recommender Systems are part of almost every web platform offering items to users. These systems are build to help people find items they like but might not discover on their own, such as movies, products, or articles. Recommenders analyze patterns of user behavior to suggest items similar to what users have liked in the past or what similar users have enjoyed. Evidence supporting the effectiveness of recommender systems includes increased sales, higher user satisfaction, and improved engagement on platforms like Amazon, Netflix, and Spotify, where personalized recommendations significantly contribute to user experience and business success."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "52e94889",
+   "metadata": {},
+   "source": [
+    "Below we will show how to build a very simple recommender system that will provide news articles to users. The system consists of two parts:\n",
+    "    \n",
+    "   1. a content-based recommender - in this case the users' past interactions are not taken into account. The model operates via item similarity, meaning that only items similar to the context item are recommended. The recommended list would be shown to the user under the title \"Similar Articles\", and the objective is to motivate the readers to read more content.\n",
```

Comment:
I think generally a content based recommender would recommend articles that are similar to what the user interacted with. The difference is between the content recommending similar items, while the collaborative approach recommending articles that people with similar history consumed (but the user we are recommending to did not).

Reply:
You are right, I actually wanted to express that, but it ended up confusing. This sentence is indeed not necessary:
in this case the users' past interactions are not taken into account